neural_speed/convert/__init__.py

```diff
@@ -21,13 +21,12 @@
 model_maps = {"gpt_neox": "gptneox", "gpt_bigcode": "starcoder", "whisper": "whisper", "qwen2": "qwen"}
 
 
-def convert_model(model, outfile, outtype="f32", model_hub="huggingface", use_quantized_model=False):
+def convert_model(model, outfile, outtype="f32", format="NE", model_hub="huggingface", use_quantized_model=False):
```

Comment:
You should add this parameter to the convert function of the other model

Reply:
Which model are u talking about ?

Reply:
NP